README.md

```diff
@@ -2,4 +2,91 @@
 
 **NCK is a Command-Line Interface (CLI), allowing you to easily request, stream and store raw reports, from the API source to the destination of your choice.**
 
-Official documentation is available [here](https://artefactory.github.io/nautilus-connectors-kit/).
+The official documentation is available [here](https://artefactory.github.io/nautilus-connectors-kit/).
+
+## Philosophy
+
+The application is composed of **3 main components** (*implemented as Python classes*). When combined, these components act as data connectors, allowing you to stream data from a source to the destination of your choice:
+
+- [Readers](nck/readers) are reading data from an API source, and transform it into a stream object.
+- [Streams](nck/streams) (*transparent to the end-user*) are local objects used by writers to process individual records collected from the source.
+- [Writers](nck/writers) are writing the output stream object to the destination of your choice.
+
+## Available connectors
+
+As of now, the application is offering:
+ 
+### Readers
```

Comment:
Ici j'aurais bien juste mis "21 Readers" en titre et retiré la ligne en dessous ("In various areas such as Media Activation, Website Analytics & many more") puisqu'on détaille les catégories juste après. Et si possible, mettre Advertising juste après Analytics, car ce sont nos 2 catégories principales 😊 Mais bon je pinaille un peu :innocent:

Reply:
est-ce qu'on spécifie vraiment qu'il y en a 21 ? (idem pour les writers) 
--> ça oblige à mettre à jour ce chiffre aussi (chose que les gens pourraient facilement oublier haha) sans apporter beaucoup d'info finalement (comme c'est dans la doc détaillée)

mais par contre, yes, je retire en-dessous ! et je rassemble advertising et analytics, bien vu !

Reply:
Good point, c'est vrai que ça n'apporte pas grand chose ! 👍